maquetado/scss/detail.scss

```diff
@@ -0,0 +1,31 @@
+@import 'colors';
+
+.title {
+  font-size: 32px;
+  font-weight: bold;
+  margin-right: 10px;
+}
+
+.titleDescription {
+  color: $color-black;
+  font-size: 20px;
+  font-weight: bold;
+  margin-right: 10px;
+}
+
+.description {
+  color: $color-grey;
+  font-size: 20px;
+}
+
+.subtitle {
+  font-size: 24px;
+  font-weight: bold;
+}
+
+.textAndDescription {
+  display: flex;
+  justify-content: flex-start;
```

Comment:
I think this is not necessary. By default it is always set like this when you add `display: flex;`